platform-plugins.md

```diff
@@ -246,7 +246,7 @@ Next,
     }
     ```
 1. Replace the implementation in `HelloPlugin.m` with a forwarding call:
-    ```objective-c
+    ```objc
```

Comment:
Yes. I found the `objc` someplace else, but it doesn't work with out site. Was just about to change it.